### ISSUE TITLE
TypeError: ts.isIntersectionTypeNode is not a function

### ISSUE THREAD

**Issue by MatthieuLebigre** [SonarSource/SonarTS#504]
**sonarqube-scanner version:** 2.0.2<br>
**Node.js version:** 8.9.4<br>
**TypeScript version:** 2.3.2<br>

Hi,
I've an issue with Typescript version: I use Typescript 2.3.2 in my project, and a ruleSet from SonarQube that contains rules only usable for higher Typescript version. But SonarTS uses my version of Typescript.

I'm not admin on SonarQube to be able to reduce the scope on the rules set.
And I'm using sonarqube-scanner to run the analysis.

Is there a way to tell sonarqube-scanner / SonarTS to use its own version of Typescript ?

**Logs**
```
ERROR: Rule error during analysis of : F:/Atlas/GitLab/aeon/aeon-clients/src/atlas/window/visualstates/_Index.ts TypeError: ts.isIntersectionTypeNode is not a function
ERROR:     at Walker.visitNode (F:\Atlas\GitLab\aeon\aeon-clients\.scannerwork\sonarts-bundle\node_modules\tslint-sonarts\lib\rules\noUselessIntersectionRule.js:51:16)
ERROR:     at Walker.SyntaxWalker.walk (F:\Atlas\GitLab\aeon\aeon-clients\.scannerwork\sonarts-bundle\node_modules\tslint\lib\language\walker\syntaxWalker.js:24:14)
ERROR:     at Rule.AbstractRule.applyWithWalker (F:\Atlas\GitLab\aeon\aeon-clients\.scannerwork\sonarts-bundle\node_modules\tslint\lib\language\rule\abstractRule.js:31:16)
ERROR:     at Rule.applyWithProgram (F:\Atlas\GitLab\aeon\aeon-clients\.scannerwork\sonarts-bundle\node_modules\tslint-sonarts\lib\rules\noUselessIntersectionRule.js:30:21)
ERROR:     at executeRule (F:\Atlas\GitLab\aeon\aeon-clients\.scannerwork\sonarts-bundle\node_modules\tslint-sonarts\lib\runner\rules.js:35:25)
ERROR:     at rules.forEach.rule (F:\Atlas\GitLab\aeon\aeon-clients\.scannerwork\sonarts-bundle\node_modules\tslint-sonarts\lib\runner\rules.js:28:51)
ERROR:     at Array.forEach (<anonymous>)
ERROR:     at Object.getIssues (F:\Atlas\GitLab\aeon\aeon-clients\.scannerwork\sonarts-bundle\node_modules\tslint-sonarts\lib\runner\rules.js:28:11)
ERROR:     at input.filepaths.map (F:\Atlas\GitLab\aeon\aeon-clients\.scannerwork\sonarts-bundle\node_modules\tslint-sonarts\lib\runner\processRequest.js:36:41)
ERROR:     at Array.map (<anonymous>)
```

**Comment by vilchik-elena** [comment]
@MatthieuLebigre hi! which version of SonarTS do you use? This problem should be fixed in SonarTS 1.4

**Comment by MatthieuLebigre** [comment]
Indeed, we had 1.3. We upgraded to 1.5 and now it's working fine. 
Thanks for the help, you can close this issue
